src/databases/postgresql.js

```diff
@@ -0,0 +1,139 @@
+import {native} from 'pg';
+import migrateAndStart from "pg-migration";
+import config from "config";
+import log from '../log';
+
+const configs = {
+  user: config.get('postgresql.user'),
+  database: config.get('postgresql.database'),
+  password: config.get('postgresql.password'),
+  host: config.get('postgresql.host'),
```

Comment:
IIRC `config.get` throws if a key is missing, which means that currently these values must always be present - even if another DB is used.